specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/locations.json

```diff
@@ -123,6 +162,140 @@
         }
       },
       "readOnly": true
+    },
+    "BillingResponseListResult": {
+      "description": "The response for the operation to get regional billingSpecs for a subscription.",
```

Comment:
The type field must be added in every model that does not have it. In this case, type: object must be added. For your reference: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#understanding-the-importance-of-type-keyword-while-defining-model-types